frontend/src/components/bucket/BucketConfigForm.vue

```diff
@@ -104,27 +104,39 @@ const onCancel = () => {
     >
       <TextInput
         name="bucketName"
-        label="Bucket name (what other users will see)"
+        label="Bucket name *"
+        placeholder="nr-costcontainment-dev"
+        helptext="The name used for the bucket in BCBox's interface."
       />
       <TextInput
         name="bucket"
-        label="Bucket"
+        label="Bucket *"
+        placeholder="bucket0123456789"
+        helptext="A unique identifier for a bucket."
       />
       <TextInput
         name="endpoint"
-        label="Endpoint"
+        label="Endpoint *"
+        placeholder="https://exampleendpoint.gov.bc.ca/"
+        helptext="The URL that specifies how to access the bucket."
```

Comment:
i would suggest `The URL of the object storage server`

Reply:
Can do `https://*.objectstore.gov.bc.ca` as a reasonable example. Still forces users to have to change it up, but we can "reasonably" expect most urls for now to match that shape for placeholder.